docs/explanation/command-chain.rst

```diff
@@ -0,0 +1,78 @@
+.. _explanation-command-chain:
+
+Command Chain
+=============
+
+Command chain is a mechanism for running small pieces of code before the main
+application executable. The chain uses ``exec`` to transition the process from
+the current element of the chain to the next. The last element of the chain is
+always the application executable.
+
+Uses
+----
+
+Command chain may be used to modify process environment, perform initialization
+steps or anything else that makes sense in a given situation. Command chain
+elements are usually reusable across applications. Command chain elements are
```

Comment:
> Command chain elements are usually reusable across applications.

As in different `app` entries in the same snap or different snaps?  The "usually" in this sentence and the next sentence are throwing me off.  Is this referring to scripts that get re-used like desktop-launch or alsa-launch?